### PR TITLE
Testing/create lesson

### PR DESCRIPTION
**Pull request by Melizzo**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 
## What is the current behavior?

- No current behaviors
## What is the new behavior?

- Created integration tests for createLesson component
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/CreateLesson/CreateLesson.test.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter } from 'react-router-dom';
+import { render, fireEvent, getByLabelText, waitFor } from '@testing-library/react';
+import CreateLesson from './CreateLesson';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux'
+import { rootReducer } from '../reducers';
+import { addLesson } from '../mockData/mockData'
+
+const store = createStore(rootReducer, {
```
Comment (Billwilke42) [inline src/CreateLesson/CreateLesson.test.js:11]: Great work with the integration tests!

File: src/mockData/mockData.js [hunk 2]
```diff
@@ -19,4 +19,31 @@ export const lesson = {
     //   answers: ['Rocks in th ground?','A city where people live', 'a boom', 'A form of energy']
     // }
   ]
+}
+
+export const teacherLesson = {
```
Comment (Billwilke42) [inline src/mockData/mockData.js:24]: nice work on this!

**Review by Billwilke42 — COMMENTED** [review]
Great job!

File: src/CreateLesson/CreateLesson.test.js (hunk 1 above)
Comment (KyleWong2510) [reply src/CreateLesson/CreateLesson.test.js:11]: Nice work.  We should consider sad paths for future tests but i think a lot of them will be on the child components

**Review by KyleWong2510 — APPROVED** [review]

**Pull request merged**